### PR TITLE
UI Refresh

### PR DESCRIPTION
**Pull request by jordandoig**

### CODE REVIEW

**Comment by rbren** [comment]
Thanks Jordan, the code looks great! Just had a few UI tweaks to request

Can we remove the left/right padding on the warning to get it lined up:

90DC4A15-870B-402E-8592-A9EAFC64CE71


On the Fairwinds + Polairs + ... logo, can we remove the `col-` class, and instead put `max-width: 400px`?

When I shrink the width of my screen, the navbar links seem to get jumbled

Screen Shot 2020-12-09 at 4 56 58 PM

Ditto on the footer

Screen Shot 2020-12-09 at 4 57 42 PM

Can we also bump up the margin a bit on the navbar links?

**Review by rbren — COMMENTED** [review]
I think we should be using the `ul`/`li` stuff [seen here](https://getbootstrap.com/docs/4.0/components/navbar/) for the navbar links - that should take care of some of the styling issues

**Comment by rbren** [comment]
@ddakoda FYI. Once the next round of changes is in, let's set up some time for the three of us to take a look

File: pkg/dashboard/templates/navbar.gohtml [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
-        <img class="logo" src="static/images/polaris-logo.png" alt="Polaris" />
-      </a>
-      <div class="navbar-right">
+    <div class="fw-nav">
+      <div class="container p-2">
         <a href="https://fairwinds.com?source=polaris" target="_blank">
-          <img class="fw-logo" src="static/images/oss-logo.png" alt="Fairwinds" />
+          <img class="fw-logo" src="static/images/white_logo_fairwinds.svg" alt="Fairwinds" />
         </a>
+        <div class="right-section col-4 p-0 d-flex justify-content-between">
```
[the remark this answers is not in the document]
Comment (jordandoig) [reply pkg/dashboard/templates/navbar.gohtml]: Looks like the `col-` classes were the issue with the navbar/footer links getting squished together, should be all good now but we can give the `ul`/`li` pattern a shot if there are still issues tomorrow.

File: pkg/dashboard/templates/navbar.gohtml [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
-      {{ if and (not .Config.DisallowExemptions) (gt (len .Config.Exemptions) 0) }}
-      <div class="exemption-alert">
-        <i class="fas fa-exclamation"></i>
-        <span>
-          Some checks were skipped based on configured exemptions.
-          <a href="?disallowExemptions=true">Click here</a>
-          to view the report with these checks included.
-        </span>
+    </div>
+    <div class="polaris-nav sticky-top">
```
[the remark this answers is not in the document]
Comment (jordandoig) [reply pkg/dashboard/templates/navbar.gohtml]: 👍🏻 Forgot to remove that, good catch! FYI, it wasn't working for me in Firefox and bootstrap doesn't have a solution to highlight the current section

File: pkg/dashboard/templates/navbar.gohtml [hunk 3]
```diff
@@ -1,24 +1,49 @@
...
+      <div class="container p-2">
+        <div class="left-section d-flex align-items-center">
+          <a class="mr-3" href="{{ .BasePath }}">
+            <img class="logo" src="static/images/polaris_logo.svg" alt="Polaris" />
```
[the remark this answers is not in the document]
Comment (jordandoig) [reply pkg/dashboard/templates/navbar.gohtml:29]: I'm not able to reproduce, let's look at this together tomorrow

**Comment by rbren** [comment]
I was wrong on the alt text - it's just an SVG with only text :)

Can you switch it to use the fancier one? https://github.com/FairwindsOps/polaris/blob/master/pkg/dashboard/assets/images/polaris-logo.png

Other than that it looks great. Still waiting to hear on the email link, but other than that I think we're good to :shipit:

**Comment by jordandoig** [comment]
I think Dakoda wanted all the new assets to be SVGs, but I can change the logo back to the old png if you like

**Review by rbren — APPROVED** [review]

**Pull request merged**